collectors/0/samza_prod.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python
+import sys
+
+from collectors.lib.samza_metric_reporter import SamzaMetricReporter
+
+CONSUMER_GROUP_ID = "tcollector_samza_prod"
+
+KAFKA_BOOTSTRAP_SERVERS = [
```

Comment:
It's a bit weird to leak this type of stuff into a publicly forked repo. Can we use ENV variables or something?